### PR TITLE
NCD Progress Report - HBA1c chart + lab results

### PR DESCRIPTION
**Pull request by anvmn**
#551

### CODE REVIEW

**Comment by anvmn** [comment]
Selection_1393

**Comment by anvmn** [comment]
Peek 2022-12-12 23-37

**Comment by anvmn** [comment]
@mariano-dagostino please review

File: client/src/elm/Translate.elm [hunk 1]
```diff
@@ -6797,6 +6814,11 @@ translationSet trans =
                     , kinyarwanda = Just "Nta bwandu afite"
                     }
 
+                LabResultsHbA1c _ ->
+                    { english = "Bellow 6%"
```
Comment (mariano-dagostino) [inline client/src/elm/Translate.elm]: @anvmn `Below (6%)`

**Review by mariano-dagostino — APPROVED** [review]
@anvmn Looks good, minor typo.

**Pull request merged**